lib/stripe/api_operations/update.rb

```diff
@@ -47,6 +43,12 @@ def serialize_params(obj)
             update_hash[k] = serialize_params(obj_values[k])
           end
 
+          obj_values.each do |k, v|
+            if v.is_a?(StripeObject) || v.is_a?(Hash)
```

Comment:
Isn't this going to include all StripeObjects and Hashes in every update? Don't we only want to include them if they've changed?